src/test/java/com/puppycrawl/tools/checkstyle/DefaultConfigurationTest.java

```diff
@@ -39,6 +67,15 @@ public void testRemoveChild() {
         assertEquals("Invalid children count", 0, config.getChildren().length);
     }
 
+    @Test
+    public void testMessage() {
```

Comment:
same.

Reply:
same. It tests both setMessage and getMessage.

Reply:
Please use long name

Reply:
done